src/android/aidl/types/file-descriptor.md

```diff
@@ -0,0 +1,38 @@
+# Sending Files
+
+Files can be sent between Binder clients/servers using the
```

Comment:
Not sure if it should go here or in the notes, but it might be worth mentioning: other kinds of FDs can be wrapped, like TCP, UDP and UNIX sockets (e.g. https://doc.rust-lang.org/stable/std/os/fd/struct.OwnedFd.html#impl-From%3CUnixStream%3E-for-OwnedFd). Of particular interest on Android: shared memory buffers called DMA-BUFs.

Reply:
Good idea, comment added.